index.js

```diff
@@ -1 +1,99 @@
-// implement your API here
+const express = require('express');
+
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+
+
+//Post
+server.post('/api/users', (req, res) => {
+    const { name, bio, created_at, updated_at } = req.body;
+    if (!name || !bio) {
+        return res.status(400).json({ errorMessage: 'Please provide name and bio for the user.' })
+    }
+    db
+        .insert({
+            name,
+            bio,
+            created_at,
+            updated_at
+        })
+        .then(res => {
```

Comment:
```suggestion
        .then(user => {
```
You already have use of res on line 11 and keeping the same name will cause you to keep going to catch block even on success